### DISCUSSION TITLE
The disk usage of MINIO cluster exceeds expectations

### DISCUSSION THREAD

**Discussion by sjgllgh** [minio/minio#20235] [Q&A]
I deployed a MINIO cluster using 10 machines through Docker, with each machine utilizing a 150GB drive. Theoretically, I should be able to store more than 500GB of content on this MINIO cluster, but in reality, the actual storage capacity falls far short of this expectation.

Minio Version: RELEASE.2023-03-24T21-41-23Z
The service startup command is: server http://node1/data -address "9000"

The Minio console shows that 100GB of content has been stored, with 14.9M objects.The actual usage scenario is as follows:

|  Node    | Driver Num | Total Memory | Used Memory |
| -------- | --------   | --------     | --------    |
| Node1    | 1          | 150G         |   97G       |
| Node2    | 1          | 150G         |   97G       |
| Node3    | 1          | 150G         |   97G       |
| ......   | ......     | ......       |   ......    |
| Node10   | 1          | 150G         |   97G       |

I don't know why the actual capacity utilized by each machine is almost approaching the actual file size stored. Are there any parameters that I can optimize in my MinIO cluster?

**Comment by klauspost** [comment]
You are just deploying independent instances. Here is how to deploy a cluster: https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html
